CONTRIBUTING.md

```diff
@@ -130,6 +131,79 @@ tackle it in a reasonable amount of time in a subsequent pull request. If you
 can't get to it soon, please create an issue and link to it from the pull
 request comment so that we don't collectively forget.
 
+## Signing your commits
```

Comment:
SIG Contributor Experience has been working on a template for the CONTRIBUTING.md and they created language for the DCO part that I think is pretty good. It explains what it used for and how to get yourself out of a pickle if you forget to do it.

https://docs.google.com/document/d/1XZ4CPHSqoJcb3KMgKyr7NAHPGibm54T5Lo-H50QLjzc/edit#

```markdown
Licensing is important to open source projects. It provides some assurances that the software
will continue to be available based under the terms that the author(s) desired. We require that
contributors sign off on commits submitted to our project's repositories. The 
[Developer Certificate of Origin (DCO)](https://developercertificate.org/) is a way to certify that 
you wrote and have the right to contribute the code you are submitting to the project.

You sign-off by adding the following to your commit messages:

    This is my commit message

    Signed-off-by: Your Name <your.name@example.com>

Git has a `-s` command line option to do this automatically:

    git commit -s -m 'This is my commit message'

If you forgot to do this and have not yet pushed your changes to the remote repository, you can 
amend your commit with the sign-off by running 

    git commit --amend -s

```

You had a really good point in your version that the author name needs to match the signoff. Maybe we can merge the two? Though I do prefer not including the text of the DCO and linking to it instead. I don't want us to have stale copies of it anywhere and it makes it harder for a contributor to get to the important bits, how to do the git commands.

What do you think?

Reply:
I like it!  Agreed that inserting the full DCO is probably not a good idea 👍  I think they're merged satisfactorily now; PTAL when convenient.